pallets/staking/src/lib.rs

```diff
@@ -2721,9 +2715,7 @@ impl<T: Trait> Module<T> {
                     0
                 });
 
-            let era_length = session_index
-                .checked_sub(current_era_start_session_index)
-                .unwrap_or(0); // Must never happen.
```

Comment:
Should the comment be kept?

Reply:
I don't think the comment made much sense to being with. If it "Must never happen." why not panic?.

Reply:
Generally speaking, we can never have a panic in the runtime. The blockchain must never stop. We should return errors and deal with them to the best of our ability instead of panicking and giving up :).

Anything that may cause a panic is a no go for the runtime.